src/main/java/org/jabref/logic/net/URLDownload.java

```diff
@@ -96,18 +99,13 @@ public URLDownload(String source) throws MalformedURLException {
     public URLDownload(URL source) {
         this.source = source;
         this.addHeader("User-Agent", URLDownload.USER_AGENT);
-    }
 
-    /**
-     * @param socketFactory trust manager
-     * @param verifier      host verifier
-     */
-    public static void setSSLVerification(SSLSocketFactory socketFactory, HostnameVerifier verifier) {
         try {
-            HttpsURLConnection.setDefaultSSLSocketFactory(socketFactory);
```

Comment:
I think this will have an impact of using custom SSL certifactes now

Reply:
I checked. --> TrustStore is created at org.jabref.logic.net.ssl.TrustStoreManager#configureTrustStore.

I created JavaDoc for it: https://github.com/JabRef/jabref/pull/11852/commits/300b7b3ff4ee55065f7c2e9b3f08ecd0fade027b

Reply:
Ah, maybe, this only works if we do this setting?! - I am not sure. How can we test this?

Reply:
From what I see it should be enough if we set in the truststore manager

Reply:
I was about to revert the change ^^. -- Since its an alpha, we can take the risk?